tools/dragen-transcriptome/4.0.3/dragen-transcriptome__4.0.3.cwl

```diff
@@ -103,12 +99,33 @@ inputs:
       CSV file that contains a list of FASTQ files
       to process. read_1 and read_2 components in the CSV file must be presigned urls.
     type: File?
+    inputBinding:
+      loadContents: true
+      prefix: "--fastq-list="
+      separate: False
+      valueFrom: "$(get_fastq_list_csv_path())"
   # Option 2:
   fastq_list_rows:
     label: fastq list rows
     doc: |
       Alternative to providing a file, one can instead provide a list of 'fastq-list-row' objects
     type: ../../../schemas/fastq-list-row/1.0.0/fastq-list-row__1.0.0.yaml#fastq-list-row[]?
+    inputBinding:
+      prefix: "--fastq-list="
+      separate: False
+      valueFrom: "$(get_fastq_list_csv_path())"
+  # Option 3
+  bam_input:
+    label: bam input
+    doc: |
+      Input a normal BAM file for the variant calling stage
```

Comment:
normal BAM file?

Reply:
Updated in https://github.com/umccr/cwl-ica/pull/347/commits/723a29ef3e0254e6288592eb31dd0d0e639f0209